src/components/schedule-visualizer/spinner.tsx

```diff
@@ -0,0 +1,56 @@
+import { makeStyles } from '@material-ui/core';
```

Comment:
I don't think this is used anymore, if we ever need it next time it is probably better to use a third party implementation like the one from material ui.

Reply:
Yeah, I'll remove this. 

FYI It is a little tricky to use the spinner inside the layer. Material-ui indeed uses a circular vector to draw the spinner but it is contained in a <div> element, which complicates using them inside the layer.

Reply:
Done!